react/components/PaymentSystemIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import flags from '../flags'
+
+const PaymentSystemIcon: React.FC<{ paymentSystem?: string }> = ({
+  paymentSystem,
+}) => (
+  <img
+    src={paymentSystem ? flags[paymentSystem] : flags.NewPaymentMethod}
+    alt=""
```

Comment:
No alt value?

Reply:
payment flag is a svg, I'll convert it to a regular react component and add an `aria-label` there, what do you think?

Reply:
actually, this component isn't being used so I'll just remove it